src/enclave/rpc_sessions.h

```diff
@@ -442,25 +492,34 @@ namespace ccf
     }
 
     std::shared_ptr<ClientEndpoint> create_client(
-      std::shared_ptr<tls::Cert> cert)
+      const std::shared_ptr<tls::Cert>& cert,
+      ccf::ApplicationProtocol app_protocol = ccf::ApplicationProtocol::HTTP1)
```

Comment:
Good point. I'm not sure the ACME server/JWT endpoint actually support HTTP/2? If so, this can be something we do once the two implementations are merged (i.e. pick HTTP/2 or HTTP/1 based on ALPN).